compiler/include/baseAST.h

```diff
@@ -188,7 +192,7 @@ enum AstTag {
 };
 
 static inline bool isExpr(AstTag tag)
-{ return tag >= E_SymExpr        && tag <= E_ExternBlockStmt; }
+{ return tag >= E_Thunk          && tag <= E_ExternBlockStmt; }
```

Comment:
If you move E_Thunk next to E_TemporaryConversionSymbol as I recommended above, you will need to undo this change.